lib/element.rb

```diff
@@ -118,13 +118,26 @@ def value
   end
 
   def click
-    Log.debug("Clicking on #{self}")
+    Log.debug("[GRIDIUM::Element] Clicking on #{self}")
+    click_retry = 2
     if element.enabled?
       ElementExtensions.highlight(self) if Gridium.config.highlight_verifications
       $verification_passes += 1
-      element.click
+      begin
```

Comment:
we might be able to have the same behavior more idiomatically here via something like this:
```
Wait.new(:timeout => 3, :interval => 1, :ignore ClickablePointException, :message => "[GRIDIUM::Element] Cannot click on element. ")
wait.until {element.click}
```
I am guessing at that actual exception name, and I think it might need to be the fully qualified name also.